content/release-notes/2023-08-28-console.md

```diff
@@ -0,0 +1,7 @@
+### Fixes & improvements
+
+- Control Plane: The [Availability Checker](/docs/reference/glossary#availability-checker) now uses the smallest Neon compute size (.25 Compute Units) to verify that a project's computes can start and read and write data. This change allows the Availability Checker to use pre-started computes more often, which shortens availability check operation time and reduces the impact of availability checks on infrastructure and compute costs.
+- Control Plane: Removed a write check that was performed on a read-only compute instance. A write check is only necessary for read-write compute instances.
```

Comment:
This line can be removed altogether IMO.

This one was a fix for a bug that reproduced in some rare cases, like project migrations between pageservers via our internal Admin API, most probably not visible to our customers at all.